src/pages/interviews.tsx

```diff
@@ -129,10 +133,21 @@ function Applicants({ type, applicants, interviews, refetchInterviews,
         </Tr>
       </Thead>
       <Tbody>
-        {applicants.map(a => 
+        {applicants
+          .sort((a1, a2) => {
+            const comp = (!sources[a1.id] && sources[a2.id]) ? 1 :
+              (sources[a1.id] && !sources[a2.id]) ? -1 :
```

Comment:
these two lines are not needed. Simply `const comp = toPinyin(...).localeCompare(toPinyin(...))` suffices